File: env.py
import os
import sys

def print_help():
    print("Usage: env [options]")
    print("Options:")
    print("  -h    Show this help")
    print("")
    print("Display environment variables.")

def main():
    args = sys.argv[1:]
    
    if any(arg in ('-h', '--help', '/?') for arg in args):
        print_help()
        return 0
    
    env_vars = os.environ
    
    # Sort environment variables alphabetically
    for key in sorted(env_vars.keys()):
        print(f"{key}={env_vars[key]}")
    
    return 0

if __name__ == "__main__":
    sys.exit(main())

File: env.js
function print_help() {
    console.log("Usage: env [options]");
    console.log("Options:");
    console.log("  -h    Show this help");
    console.log("");
    console.log("Display environment variables.");
}

const args = process.argv.slice(2);

if (args.includes('-h') || args.includes('--help')) {
    print_help();
    process.exit(0);
}

const envVars = process.env;
const sortedKeys = Object.keys(envVars).sort();

for (const key of sortedKeys) {
    console.log(`${key}=${envVars[key]}`);
}